src/main/python/systemds/utils/helpers.py

```diff
@@ -15,34 +15,16 @@
 # ------------------------------------------------------------------------------
 
 import os
-import platform
 import subprocess
-from fnmatch import fnmatch
 from itertools import chain
-from typing import List, Iterable, Dict
+from typing import Iterable, Dict
 from importlib.util import find_spec
 
 from py4j.java_gateway import JavaGateway
-
-__all__ = ['get_gateway', 'create_params_string']
+from py4j.protocol import Py4JNetworkError
 
 JAVA_GATEWAY = None
 MODULE_NAME = 'systemds'
-# TODO remove standalone
-JAR_FILE_STANDALONE = 'systemds-*-standalone.jar'
-JAR_FILE_NAMES = ['systemds-*-SNAPSHOT.jar', 'systemds-*.jar', JAR_FILE_STANDALONE]
-AUTO_START_JMLC_SERVER = True
-
-
-def set_auto_start_jmlc(should_start_server: bool) -> None:
-    """
-    Can activate/deactivate automatic start of JMLC API Gateway.
-    Has to be called before any communication took place (`.compute()` was called).
-
-    :param should_start_server: activate/deactivate
-    """
-    global AUTO_START_JMLC_SERVER
-    AUTO_START_JMLC_SERVER = should_start_server
```

Comment:
I removed this and switched to on demand auto starting of jmlc

Reply:
:+1: